app/views/admin/secrets/index.html.erb

```diff
@@ -1,11 +1,10 @@
-
-  <% if !@vault_enabled %>
-    <div class="crayons-notice crayons-notice--info mb-6" role="alert">
-      <p>
-        Vault is not currently setup for your application. This means your secrets are being stored as ENV variables and cannot be updated here. To update them you will need to update the appropriate files in your environment.
-      </p>
-    </div>
-  <% end %>
+<% if !@vault_enabled %>
```

Comment:
Even if it's unrelated, we're already changing it so might as well change it properly:

```suggestion
<% unless @vault_enabled %>
```